assets/js/util/tracking/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Event tracking utilities.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { bootstrapTracking } from './index.private';
+export {
+	disableTracking,
+	enableTracking,
+	isTrackingEnabled,
+	toggleTracking,
+	trackEvent,
+} from './index.private';
```

Comment:
Can we instead declare the public interface in this file, similar like we're doing in the other new modules (e.g. `googlesitekit/api` and `googlesitekit/api/cache`)? This here works from an interface standpoint, but these functions are the public part of the module and therefore shouldn't be declared in the `.private` file.

We'll need to export some more variables from `index.private.js` then, but that'd be alright.